src/renderer/src/lib.rs

```diff
@@ -12,15 +12,14 @@ extern crate mopa;
 
 extern crate glutin;
 extern crate cgmath;
-
-extern crate amethyst_ecs;
+extern crate specs;
 
 /// Contains the included Render Targets
 pub mod target;
 /// Contains the included Passes
 pub mod pass;
 
-use amethyst_ecs::{Component, VecStorage};
+use self::specs::{Component, VecStorage};
```

Comment:
Didn't notice this before, but why are we implementing `Component` and `VecStorage` over here? Since the renderer is meant to be usable outside of the ECS, this functionality should be moved up to the top-level crate under `amethyst::ecs`.